src/main/java/karuna/karuna_backend/post/infrastructure/PostController.java

```diff
@@ -35,4 +35,16 @@ class PostController {
     PostDto createPost(@RequestBody PostCreateDto postCreateDto) {
         return postService.createPost(postCreateDto);
     }
+
+    @Operation(summary = "Get posts with pagination", description = "Fetches a paginated list of posts")
+    @ApiResponse(responseCode = "200",
+            description = "Posts fetched successfully",
+            content = @Content(mediaType = "application/json", schema = @Schema(implementation = PostWrapper.class)))
+    @ApiResponse(
+            responseCode = "403",
```

Comment:
What with 401?

Reply:
I addded 401. I deleted 403 because this endpoint is for both - Admin and User.